.github/workflows/e2e-tests.yml

```diff
@@ -91,15 +91,21 @@ jobs:
         if: inputs.test_type != 'cctp'
         uses: OffchainLabs/actions/run-nitro-test-node@a20a76172ce524832ac897bef2fa10a62ed81c29
         with:
-          nitro-testnode-ref: aab133aceadec2e622f15fa438f6327e3165392d
-          l3-node: ${{ matrix.test.type == 'orbit' }}
-          no-l3-token-bridge: ${{ matrix.test.type != 'orbit' }}
+          nitro-testnode-ref: badbcbea9b43d46e115da4d7c9f2f57c31af8431
```

Comment:
It's a newer tag that funds the wallet with native currency on L2. You can see the change here:
https://github.com/OffchainLabs/nitro-testnode/commit/badbcbea9b43d46e115da4d7c9f2f57c31af8431